symfony/client/src/orangehrmAuthenticationPlugin/pages/ResetPassword.vue

```diff
@@ -0,0 +1,123 @@
+  - if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+  - Boston, MA  02110-1301, USA
+  -->
+
+<template>
+  <div class="orangehrm-forgot-password-container">
+    <div class="orangehrm-forgot-password-wrapper">
+      <div class="orangehrm-card-container">
+        <oxd-form method="post">
+          <oxd-text tag="h6" class="orangehrm-forgot-password-title">
+            Reset password
+          </oxd-text>
+          <oxd-divider />
+          <card-note
+            note-text="Set a new Password"
+            class="orangehrm-forgot-password-card-note"
+          />
+          <oxd-form-row>
+            <oxd-input-field
+              :model-value="username"
+              name="username"
+              label="Username"
+              label-icon="person"
+              disabled
+            />
+          </oxd-form-row>
+          <oxd-form-row>
+            <oxd-input-field
+              v-model="user.newPassword"
+              name="password"
+              label="New Password"
+              label-icon="key"
+              placeholder="password"
+              type="password"
+              :rules="rules.newPassword"
+              autocomplete="off"
+              required
+            />
+          </oxd-form-row>
+          <oxd-form-row>
+            <oxd-input-field
+              v-model="user.confirmPassword"
+              name="username"
+              label="Confirm Password"
+              label-icon="key"
+              placeholder="password"
+              type="password"
+              :rules="rules.confirmPassword"
+              autocomplete="off"
+              required
+            />
+          </oxd-form-row>
+          <oxd-divider />
+          <div class="orangehrm-forgot-password-button-container">
+            <oxd-button
+              class="orangehrm-forgot-password-button"
+              display-type="secondary"
+              size="large"
+              label="Reset Password"
+              type="submit"
+            />
+          </div>
+        </oxd-form>
+      </div>
+    </div>
+    <slot name="footer"></slot>
+  </div>
+</template>
+
+<script>
+import CardNote from '../components/CardNote';
+import {checkPassword} from '@ohrm/core/util/helper/password';
+import {
+  required,
+  shouldNotExceedCharLength,
+} from '@ohrm/core/util/validation/rules';
+
+export default {
+  name: 'ResetPassword',
+  components: {
+    'card-note': CardNote,
+  },
+  props: {
+    username: {
+      type: String,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      user: {
+        username: '',
+        newPassword: '',
+        confirmPassword: '',
+      },
+      rules: {
+        newPassword: [required, shouldNotExceedCharLength(64), checkPassword],
+        confirmPassword: [
+          required,
+          shouldNotExceedCharLength(64),
+          v => (!!v && v === this.password) || 'Passwords do not match',
```

Comment:
You may have changed the variable and forgot to update here
```suggestion
          v => (!!v && v === this.user.newPassword) || 'Passwords do not match',
```